xwiki-platform-core/xwiki-platform-oldcore/src/test/java/com/xpn/xwiki/pdf/impl/PdfExportImplTest.java

```diff
@@ -128,6 +131,7 @@ public void applyCSSWhenExistingStyleDefinedUsingShorthandNotation() throws Exce
             + "</div>"
             + "</body></html>";
 
-        assertEquals(expected, pdfExport.applyCSS(html, css, xcontext));
+        assertEquals(expected.replaceAll(">\\s+<", "><"),
```

Comment:
Could you explain this?

Reply:
ok just seen the explanation in the PR. I still think the code requires a comment, it's far from obvious and we need to also understand why the behavior change is not a problem for backward compatibility.

Reply:
I'll prepare a new version of the patch with a comment (and removing the `http://xml.org/sax/features/namespaces`) if you do not deem it necessary.